index.js

```diff
@@ -1,3 +1,9 @@
 function subtract(a, b) {
   return a - b;
-}
\ No newline at end of file
+}
+
+const sum =(a  ,b) => {
```

Comment:
Oi Edu, 

Vamos usar por padrão neste projeto apenas function, ao invés de arrow function. (joinha)
Ai o código ficaria assim:
```suggestion
function sum(a, b) {
return a + b;
}

```

Reply:
Você comentou apenas a linha de cima, mas irei utilizar sua sugestão obrigado!